internal/gittuf-git/git.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// gitCmd represents the base command when called without any subcommands
+var gitCmd = &cobra.Command{
```

Comment:
How much control do we have over what flags package ends up being used by cobra? Do we have to resort to a go mod replace directive? I ask because cobra can be a bit tetchy about unrecognized args, and I'm not yet convinced cobra.ArbitraryArgs is ok. Do we want to just have a very lightweight wrapper around the stdlib flags library? Do we process flags ourselves?


I don't have answers to these questions yet but I'd like the both of you to think / explore a bit. :smile: